tests/test_notifications.py

```diff
@@ -262,7 +261,7 @@ def test_node_subscriptions_and_backrefs_removed_when_node_is_deleted(self):
             subscription.save()
 
             s = getattr(project.creator, 'email_transactional', [])
-            assert_equal(len(s), 1)
+            assert_equal(len(s), 2)
```

Comment:
Yes! So each `notificationsubscription` carries [three lists](https://github.com/CenterForOpenScience/osf.io/blob/711558a79268f0b72630083b3c2a70d97875fe9e/website/notifications/model.py#L22-L24), which ... probably indicate whom to send emails to when, ya?

![screen shot 2016-02-23 at 2 07 10 pm](https://cloud.githubusercontent.com/assets/134455/13263266/c0c3b938-da36-11e5-8486-ddb338a3339f.png)